tests/test-extensions/metadata-extension/src/main/java/org/mule/test/metadata/extension/resolver/TestInputResolver.java

```diff
@@ -11,9 +11,12 @@
 import org.mule.runtime.api.metadata.MetadataContext;
 import org.mule.runtime.api.metadata.MetadataResolvingException;
 import org.mule.runtime.api.metadata.resolving.InputTypeResolver;
+import org.mule.runtime.api.metadata.resolving.MetadataComponent;
 
 public class TestInputResolver implements InputTypeResolver<String> {
 
+  public static String INPUT_RESOLVER_NAME = MetadataComponent.INPUT.name();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "INPUT_RESOLVER_NAME" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3008))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7668&issues=AWjdfEJ1otIXnmsUGfbI&open=AWjdfEJ1otIXnmsUGfbI)

[ISSUE_KEY]: AWjdfEJ1otIXnmsUGfbI